src/api/v3/liquidity.ts

```diff
@@ -0,0 +1,235 @@
+import BigNumber from "bignumber.js";
+import { Tick, Pool, Price } from "@plenty-labs/v3-sdk";
+import Config from "../../config/config";
+import { ContractStorage, getRealPriceFromTick } from "./helper";
+import { store } from "../../redux";
+
+export const connectedNetwork = Config.NETWORK;
+
+export const calculateCurrentPrice = async (
+  tokenXSymbol: string,
+  tokenYSymbol: string,
+  refernceToken: String
+): Promise<any> => {
+  try {
+    let currentPrice;
+
+    let contractStorageParameters = await ContractStorage(tokenXSymbol, tokenYSymbol);
+    let PoolObject = new Pool(
```

Comment:
pool object is redundant, square root price is already there in contract storage